css/styles.css

```diff
@@ -5,8 +5,10 @@ table {
   border-collapse: collapse;
   margin-bottom: 20px;
   font-family: Arial, sans-serif;
-  background-color: #222; /* Dark background color */
```

Comment:
Did a prettify js beautify tool maybe come through here? 

I couldn't find a definitive guide on CSS best practices, but the MDN docs [mention how](https://developer.mozilla.org/en-US/docs/Web/CSS/Comments) to do inline comments and don't say not to use them. It's a bit of a nit, but I feel that they are more readable on the same line (or if on different lines, above might be clearer)

Reply:
It looks like VSCode did some prettification, but my newest re-installation doesn't have that problem! So it's reverted.

Looking at it now, I agree that the way it "prettified" was kinda terrible.